crates/sui-graphql-e2e-tests/tests/tests.rs

```diff
@@ -16,7 +16,7 @@ datatest_stable::harness!(run_test, TEST_DIR, r".*\.(mvir|move)$");
 #[cfg_attr(msim, msim::main)]
 async fn run_test(path: &Path) -> Result<(), Box<dyn std::error::Error>> {
     telemetry_subscribers::init_for_testing();
-    if cfg!(feature = "pg_integration") {
+    if !cfg!(msim) {
```

Comment:
Can you add a comment on this `if`? Don't quite understand it.

Reply:
Yeah, essentially these tests don't really work under msim since they rely on external things like a database so this shuts them off during those tests.